plugin/evm/mempool.go

```diff
@@ -21,7 +21,12 @@ const (
 	discardedTxsCacheSize = 50
 )
 
-var errNoGasUsed = errors.New("no gas used")
+var (
+	errTxAlreadyKnown = errors.New("tx already known")
```

Comment:
why do we add this error then bypass it at all the callsites?

Reply:
You're totally right - this should be returned from `Add`.

Reply:
🦅 👁️

Reply:
Yeah this is supposed to be returned from `Add` to indicate that a duplicate existed...